tests/Integration/RequestIntegrationTest.php

```diff
@@ -39,7 +39,7 @@ public function testInvokeWithRequestHavingIpAddress(bool $shouldSendPii, array
         $event = new Event();
         $event->getUserContext()->setData(['foo' => 'bar']);
 
-        $request = new ServerRequest(['REMOTE_ADDR' => '127.0.0.1']);
+        $request = new ServerRequest('GET', new Uri(''), [], null, '1.1', ['REMOTE_ADDR' => '127.0.0.1']);
```

Comment:
For coherence with the rest of the code, can you please create a valid `Uri` object?